operator/src/main/java/org/keycloak/operator/v2alpha1/crds/KeycloakSpec.java

```diff
@@ -16,9 +16,16 @@
  */
 package org.keycloak.operator.v2alpha1.crds;
 
+import io.fabric8.kubernetes.api.model.PodTemplate;
+
+import java.util.Map;
+
 public class KeycloakSpec {
 
     private int instances = 1;
+    private String image;
+    private Map<String, String> serverConfiguration;
+    private PodTemplate podTemplate;
```

Comment:
I would suggest to move this out to another PR

Reply:
👍